### PR TITLE
Upgrade to hawkular-parent 16 - upgarde to WildFly 9

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: rest-servlet/src/main/java/org/hawkular/inventory/rest/RestFilters.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
```
Comment (metlos) [inline rest-servlet/src/main/java/org/hawkular/inventory/rest/RestFilters.java]: Is this file supposed to be part of this PR?

**Comment by ppalaga** [comment]
@metlos sorry based on an improper branch, sorry.

**Comment by ppalaga** [comment]
e8006a2 should pass CI as I removed the outdated `jboss-javaee-7.0-with-resteasy` import from rest-api

**Comment by ppalaga** [comment]
@metlos could you please merge?

**Pull request merged**